src/backend/vulkan/src/command.rs

```diff
@@ -749,6 +749,42 @@ impl com::RawCommandBuffer<Backend> for CommandBuffer {
             )
         }
     }
+
+    fn push_compute_constants(
+        &mut self,
+        layout: &n::PipelineLayout,
+        stages: pso::ShaderStageFlags,
```

Comment:
Should be always `COMPUTE` so we don't need this.

Reply:
Ah, right.